assets/app/scripts/directives/oscFileInput.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('oscFileInput', function(Logger) {
+    return {
+      restrict: 'E',
+      scope: {
+        name: "@",
+        model: "=",
```

Comment:
@jwforres this is supposed to reflect that this is our version of '<input type=file >'.  Do we really want to call it 'filecontent'?